### ISSUE TITLE
"Take a screenshot" does not include the whole page.

### ISSUE THREAD

**Issue by MrCoder** [chinchang/web-maker#299]
It seems that "take a screenshot" only capture the visible part of the preview panel. It would be much better if it capture the whole page.

**Comment by chinchang** [comment]
You mean along with the code and other interface also?

**Comment by MrCoder** [comment]
No. I mean when the page is longer than the view port (the visible part of
the page that falls in the window).
